apps/files/src/components/File-Details.vue

```diff
@@ -1,131 +1,130 @@
 <template lang="pug">
-	// material icon mixin
```

Comment:
Please fix the whitespace Messer. Thx

Reply:
Fixed. File-Details.vue represents the same indentation in the master branch except for the lines I changed.